### DISCUSSION TITLE
A320neo screens does not open

### DISCUSSION THREAD

**Discussion by McGee122** [flybywiresim/aircraft#2427] [Help]
I did all of things in the checklist but ı couldn't open the screens ı upload msfs2020 again and nothing changed what can ı do?
<img width="1913" alt="Ekran görüntüsü 2020-12-14 152457" src="https://user-images.githubusercontent.com/75931636/102085734-16f98f80-3e28-11eb-9ef7-654437bfcfd5.png">

**Comment by MapleNAV-TCASFAULT** [comment]
You're running the default a320neo, I'm guessing you didn't use the installer if so;
Link: https://flybywiresim.com/a32nx (press the installer, also select the dev version for latest content :) )

**Reply by McGee122** [reply]
ı download with that it's but again have problem

**Reply by MapleNAV-TCASFAULT** [reply]
Try clearing out your community folder/moving stuff out to a temp folder and try again. As there may be conflicting mods or liveries!

**Reply by McGee122** [reply]
it didn't changed anything

**Reply by MapleNAV-TCASFAULT** [reply]
Do you have a screenshot of your community folder? Just scratch out some personal info if you're gonna publish it. Also make sure you have the default a320neo installed in msfs 2020, you can find out in content manager in profile in msfs

**Reply by McGee122** [reply]
ı can't send you screenshot now tomorrow  or this night  I can send because ı am downloading MSFS2020 again

**Reply by MapleNAV-TCASFAULT** [reply]
I saw your question in #help on discord. So like ThatRedMelon said, make sure that you have the latest version of the mod, as older versions are no longer compatible with the current msfs version. Another way to get the installer is to use the command " .installer " in the FBW discord.

**Comment by LucasCano** [comment]
What release had you installed? I have the same problem with release 0.5.0 and 0.5.1
I have installed 0.4.2  panel lights turn on after  several seconds, but some funtions on the MCDU does not work well.

**Comment by Inaxair** [comment]
You can download the latest dev version. Also, make sure to have the latest version of the simulator, and have battery power or any electrical supply
